#! /bin/sh /usr/share/dpatch/dpatch-run
## debian-paths.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: Fix default path's to match the ones used in Debian

@DPATCH@
diff -urNad freepbx-module-phpagiconf~/page.phpagiconf.php freepbx-module-phpagiconf/page.phpagiconf.php
--- freepbx-module-phpagiconf~/page.phpagiconf.php	2007-04-02 11:21:33.000000000 +0300
+++ freepbx-module-phpagiconf/page.phpagiconf.php	2007-04-23 13:25:54.000000000 +0300
@@ -23,10 +23,10 @@
 $asman_server = isset($_REQUEST['asman_server'])?$_REQUEST['asman_server']:'localhost';
 $asman_port = isset($_REQUEST['asman_port'])?$_REQUEST['asman_port']:'5038';
 $asmanager = isset($_REQUEST['asmanager'])?$_REQUEST['asmanager']:''; // This comes from the API module
-$cepstral_swift = isset($_REQUEST['cepstral_swift'])?$_REQUEST['cepstral_swift']:'/opt/swift/bin/swift';
+$cepstral_swift = isset($_REQUEST['cepstral_swift'])?$_REQUEST['cepstral_swift']:'/usr/bin/swift';
 $cepstral_voice = isset($_REQUEST['cepstral_voice'])?$_REQUEST['cepstral_voice']:'David';
 $setuid = isset($_REQUEST['setuid'])?$_REQUEST['setuid']:'0';
-$basedir = isset($_REQUEST['basedir'])?$_REQUEST['basedir']:'/var/lib/asterisk/agi-bin/';
+$basedir = isset($_REQUEST['basedir'])?$_REQUEST['basedir']:'/usr/share/asterisk/agi-bin/';
 
 $dispnum = "phpagiconf"; //used for switch on config.php
 
